### PR TITLE
fix: change starting position of dome lights on runway

### PR DESCRIPTION
**Pull request by Awemeter**
Fixes #[issue_no]

## Summary of Changes

Fixes dome light position when spawning on the runway.
## Additional context

Discord username (if different from GitHub): Awemeter#7919

## Testing instructions

Spawn on runway at any airport and check that the switch and the lights for the dome lights are off.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by pepperoni505 — APPROVED** [review]
LGTM

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -93,6 +93,7 @@
 1. [MCDU] Fixed center runway id displaying on the flightplan page - @SergeySoprykin (SergeyS)
 1. [FCU] Added custom FCU font - @tyler58546 (tyler58546)
 1. [FLIGHTMODEL] Quick fix to flap lift for MSFS update 1.13.16.0 - @donstim (donbikes#4084)
+1. [LIGHTS] Fixed runway spawn postition for dome light  - @Awemeter (Awemeter#7919)
```
Comment (tracernz) [inline .github/CHANGELOG.md:96]: s/postition/position/

**Comment by tracernz** [comment]
Is it normal to takeoff with the dome light off? The dome light is the only light that runs off battery and my understanding is that it should always be on dim during takeoff in case of electrical failure, then switched off after as required. Maybe something for #pilotfeedback.

**Comment by Awemeter** [comment]
Snap you do not edit the runway.flt tho. Also I just asked Dan and he said you would keep them off on take off.

**Comment by Snapmatics** [comment]
> 
> 
> Snap you do not edit the runway.flt tho. Also I just asked Dan and he said you would keep them off on take off.

I didn't edit the runway.FLT.

**Comment by Snapmatics** [comment]
> 
> 
> Snap you do not edit the runway.flt tho. Also I just asked Dan and he said you would keep them off on take off.

I think you mentioned the wrong person btw haha

**Comment by Awemeter** [comment]
I just asked Dan and he said you would keep them off on take off.

**Comment by starvingjavi** [comment]
Name: Javi#2557
Date of testing: 18/2/2021
Version of the sim: 1.13.16.0
PR Tested: #3590 
Tier of testing: 1
Changes to observe: Dome light position when spawning on runway
Testing technique: Spawn on runway, check dome light

Steps made to test the Pull Request:

1.) Spawned on runway 10 at MMAA during the day
2.) Checked that dome light switch position was off. Check for dome lights to be off
3.) Spawned on runway 10 at MMAA during the night
4.) Checked that dome light switch position was off. Check for dome lights to be off


Issues: None

Overall Rating: LGTM

Conclusions of the testing: Everything works as described on the PR

**Comment by beheh** [comment]
@Awemeter Could you fix the CHANGELOG.md typo still please?

**Review by Snapmatics — APPROVED** [review]
Ok, the issue with the changelog has been fixed

**Pull request merged**